Dockerfile

```diff
@@ -0,0 +1,19 @@
+# use node as base image
```

Comment:
This file will be the same across all repos.  It's not that complicated but we could DRY it up by having a docker image that takes care of lines 1-16.  Thoughts?

Reply:
I think it would be cleaner to have one build image in toolkit